lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.spec.ts

```diff
@@ -1092,32 +1085,17 @@ describe('ContentNodeSelectorComponent', () => {
                     component.select.subscribe((nodes) => {
                         expect(nodes).toBeDefined();
                         expect(nodes).not.toBeNull();
-                        expect(component.chosenNode).not.toBeNull();
+                        expect(component.chosenNode[0]).not.toBeNull();
                         expect(component.isSelectionValid).not.toBeNull();
                     });
 
                     component.documentList.ready.emit(nodePage);
                     fixture.detectChanges();
                 }));
 
-                it('should NOT be null after clicking on a node in the list (onNodeSelect)', async(() => {
```

Comment:
why this unit test is not valid anymore?

Reply:
it has been copied and pasted... there is another identical in the same suite